checks/raw/binary_artifacts.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package raw
+
+// File represents a file.
+type File struct {
+	Path string
+	Ext  string
```

Comment:
nit: `Ext` implies this is the file extension, but that's not always the case right? (And if it were, there's probably not much value in storing this as a separate field when we can just derive it from the path). 

Should this be "Type" instead?

Reply:
I agree and wanted to change it. How about removing it entirely for now? If ppl ask for it, we can add it later. wdut?

Reply:
removing for now SGTM!